sparsh:
    #Publish all joint states
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 100

    #Position controllers
    top_right_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: top_right_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    top_left_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: top_left_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    middle_right_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: middle_right_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    middle_left_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: middle_left_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    bottom_right_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: bottom_right_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    bottom_left_column_to_base_link_position_controller:
        type: effort_controllers/JointPositionController
        joint: bottom_left_column_to_base_link
        pid: {p: 100.0, i: 0.1, d: 1.0}

    
